2019/03/16 14:34:54 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: frontend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "frontend.test"
2019/03/16 14:46:42 [error] 6#6: *15 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: frontend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "frontend.test"
2019/03/16 14:47:08 [error] 6#6: *15 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: frontend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "frontend.test"
2019/03/16 14:53:53 [error] 6#6: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: frontend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "frontend.test"
2019/03/16 14:53:55 [error] 6#6: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: frontend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "frontend.test"
2019/03/16 14:53:55 [error] 6#6: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: frontend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "frontend.test"
2019/03/16 14:53:56 [error] 6#6: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: frontend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "frontend.test"
2019/03/16 14:53:56 [error] 6#6: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.96.1, server: frontend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "frontend.test"
